alpakaConfig.cmake

```diff
@@ -569,6 +590,7 @@ IF(NOT TARGET "alpaka")
     IF(${_ALPAKA_INCLUDE_DIRECTORIES_PUBLIC_LENGTH} GREATER 0)
         TARGET_INCLUDE_DIRECTORIES(
             "alpaka"
+            SYSTEM
```

Comment:
What will this do?

Reply:
Adds `-isystem` to anything inside `_ALPAKA_INCLUDE_DIRECTORIES_PUBLIC`: e.g. boost was in there.

If you want to add `alpaka` includes (`-I`) *and* boost includes (`-isystem`) do it in two commands to avoid warnings from third party headers.

Reply:
In other words: I will remove `_ALPAKA_INCLUDE_DIRECTORY` from `_ALPAKA_INCLUDE_DIRECTORIES_PUBLIC` and add it explicitly so we still get *our own warnings* :)